README.md

```diff
@@ -24,11 +25,21 @@ A basic example of an RLHF interface with a Gradio app.
    huggingface.co, the app will use your token to automatically store new HITs
    in your dataset. Setting `FORCE_PUSH` to "yes" ensures that your repo will
    force push changes to the dataset during data collection. Otherwise,
-   accidental manual changes to your dataset could result in your space gettin
+   accidental manual changes to your dataset could result in your space getting
    merge conflicts as it automatically tries to push the dataset to the hub. For
    local development, add these three keys to a `.env` file, and consider setting
    `FORCE_PUSH` to "no".
+
+To launch the Space locally, run:
+
+```bash
+python app.py
+```
+
+The app will then be available at a local address, such as http://127.0.0.1:7860
```

Comment:
just added a quick wording change, because the address is not always http://127.0.0.1:7860 (sometimes it's e.g. http://127.0.0.1:7861)